lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,54 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const AllTabs = styled.div`
+    display: flex;
+    justify-content: none;
+    align-items: center;
+    flex-direction: row;
+    color: #fff;
+    background-color: #333;
+    margin: 0 5px;
+    padding: 2px 10px;
+    font-size: 12px;
+    letter-spacing: 2px;
+    cursor: pointer;
+    font-weight: bold;
+    color: black;
+
+    :hover {
+      text-decoration: underline;
+    }
+
+    ${props => (props.type === 'primary' ? `border: 1px solid red` : null)}
+    ${props => (props.type === 'active-tab' ? `background-color: black; color: white`: null)}
+`;
+
+
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
-    <div
-      className={''}
-      onClick={() => {
+    <AllTabs
+      type={props.tab === props.selectedTab ? 'primary' : 'active-tab'}
```

Comment:
`active-tab` is not being applied. This has to do with the comment above. Right now `props.selectedTab` is a function. Refactor the props and this will work perfectly.